### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BorisJegorovic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ Web application based on PHP, MySQL, and Javascript/AJAX, visually designed with
 3) Logout (Signout)
 4) Remember me
 5) Users and Remember Me database creation
+6)
```
Comment (rillian) [inline README.md]: Did you intend to include more changes here?

Comment (BorisJegorovic) [reply README.md]: Not for now. I am trying to explain Ognjan how GitHub works.

Comment (rillian) [reply README.md]: Ah, ok. Welcome Ognjan! :)

Comment (Ogiliano) [reply README.md]: Thank you, Ralph. :)

**Pull request merged**